docs/fixtures.md

```diff
@@ -88,6 +88,38 @@ class FilesSuite extends FunSuite {
 }
 ```
 
+If `beforeEach()` and `afterEach()` methods supposed to be asynchronous computations 
```

Comment:
I prefer to use active sentences like "Use X" or "Override Y" when writing docs instead of "It's possible to use X" or "You can do Y".

```suggestion
Use `AsyncFixture[T]` for fixtures that need to be loaded asynchronously. Override the `munitAsyncFixtures` method to register async fixtures.
```